docs/troubleshooting.md

```diff
@@ -1,20 +1,24 @@
+| permission denied error.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Please refer [here](https://cloud.google.com/storage/docs/gcsfuse-mount#authenticate_by_using_a_service_account) to know more about permissions.(e.g.  **Issue**:mkdir: cannot create directory ‘gcs/test’: Permission denied. User can check specific errors by enabling logs with --debug_fuse and --debug_gcs flags. **Solution**: Provide roles/storage.objectAdmin role on the bucket.)                                                                                                                                                                                                                           |
+| Bad gateway error while installing/upgrading GCSFuse:<br/> <pre>Err: http://packages.cloud.google.com/apt gcsfuse-focal/main amd64 gcsfuse amd64 1.2.0<br/>502  Bad Gateway [IP: xxx.xxx.xx.xxx 80]</pre>                                                                                                                                                                                                                                                                                                                                                                                                             | This error is seen when the url used in /etc/apt/sources.list.d/gcsfuse.list file uses HTTP protocol instead of HTTPS protocol. Run the following commands to update /etc/apt/sources.list.d/gcsfuse.list file with the https:// url.<br/> <pre>$ sudo rm /etc/apt/sources.list.d/gcsfuse.list <br/>$ export GCSFUSE_REPO=gcsfuse-`lsb_release -c -s<br/>$ echo \"deb https://packages.cloud.google.com/apt $GCSFUSE_REPO main\" &#124; sudo tee /etc/apt/sources.list.d/gcsfuse.list                                                                                                                                  |
```

Comment:
Looks like the `<pre>` tag before `sudo rm` in this line doesn't have a matching `</pre>` tag. I'm not sure if it causes an issue in markdown though.